spec/functions/datetime.md

```diff
@@ -282,11 +282,14 @@ the functions `:datetime`, `:date`, and `:time`.
   - `local`
   - `UTC`
 
-> [!NOTE]
-> The value `local` permits a _message_ to convert a date/time value
-> into a [floating](https://www.w3.org/TR/timezone/#floating) time value
-> (sometimes called a _plain_ or _local_ time value) by removing
-> the association with a specific time zone.
+The value `local` corresponds to the default time zone.
+
+If the _operand_ value does not include a time zone,
+it is presumed to use the default time zone.
```

Comment:
I think it's not clear here if "the default time zone" is meant to be the result of accessing the system time zone setting, or a single default time zone (e.g. UTC).

Reply:
The intent with the proposed language is to match the text from earlier in the same section: https://github.com/unicode-org/message-format-wg/blob/0028d177cafe85225ecb396e7570139c0a8aacf8/spec/functions/datetime.md?plain=1#L265-L268

Does that context clarify the meaning here?

Reply:
```suggestion
it is presumed to use the default time zone specified by the formatting context.
```

Ah, I missed that. How about this, then?